The action github fails with:
```
./build.sh linux
./build.sh: line 213: pcregrep: command not found
cat: write error: Broken pipe
Error: Process completed with exit code 127.
```

Before this, it not happened, probably changed something in the ubuntu image? I will add the installation of pcregrep and i will upgrade actions/checkout@v3, actions/setup-node@v3 to v4.

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -10,11 +10,11 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [16.x]
+        node-version: [20.x]
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v4
     - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v3
+      uses: actions/setup-node@v4
       with:
         node-version: ${{ matrix.node-version }}
     - name: Set output
@@ -27,7 +27,7 @@ jobs:
           echo $RELEASE_VERSION
           echo ${{ steps.vars.outputs.tag }}
     - run: git submodule update --init
-    - run: sudo apt-get update && sudo apt install libjpeg-dev ninja-build
+    - run: sudo apt-get update && sudo apt install pcregrep libjpeg-dev ninja-build
     - run: ./build.sh linux
     - run: ./build.sh emscripten
     - run: ./build.sh emscripten-simd

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -38,7 +38,7 @@ jobs:
         path: |
           packaging/opencv-4.7.0.zip
           packaging/opencv-js-4.7.0-emcc-3.1.26.zip
-          packaging/opencv-js-simd-4.7.0-emcc-3.1.26.zip
+          packaging/opencv-js-4.7.0-emcc-3.1.26-simd.zip
     - name: Release
       uses: softprops/action-gh-release@v1
       if: startsWith(github.ref, 'refs/tags/')
@@ -46,4 +46,4 @@ jobs:
        files: |
         packaging/opencv-4.7.0.zip
         packaging/opencv-js-4.7.0-emcc-3.1.26.zip
-        packaging/opencv-js-simd-4.7.0-emcc-3.1.26.zip
+        packaging/opencv-js-4.7.0-emcc-3.1.26-simd.zip

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -30,7 +30,7 @@ jobs:
     - run: sudo apt-get update && sudo apt install pcregrep libjpeg-dev ninja-build
     - run: ./build.sh linux
     - run: ./build.sh emscripten
-    - run: ./build.sh emscripten-simd
+    - run: ./build.sh --simd emscripten
     - run: echo "Done!"
     - uses: actions/upload-artifact@v3
       with:

